### ISSUE TITLE
Accessibility Issue: Keyboard focus is getting trapped inside Theme color section of Font Color and Highlighted color due to which control is not accessible using "Shift-Tab"

### ISSUE THREAD

**Issue by shashankpatel1542** [pnp/sp-dev-fx-controls-react#1944]
User Experience:
Users who depend on keyboard will be impacted if "Automatic" control is not accessible using "Shift-Tab" key after you move inside Theme color picker or Standard Color picker. Keyboard functions and people that are blind or have low vision and people that use keyboard emulators cannot access the content which will confuse them.

Please Note: Standard Color picker is not accessible with pressing tab or arrow key at all.

**Actual Result:**
"Automatic" control is not accessible using "Shift-Tab" key upon navigating inside Theme color to pick a color.
Issue Reproduce for "No Color" control inside "Highlight color" button of the same dialog.

**Expected Result:**
"Automatic" control should be accessible using "Shift-Tab" key once you navigate inside Theme color/Standard color.

![Image](https://github.com/user-attachments/assets/eb357b92-e862-4b64-a3f3-5efd92cb0ed2)

We observed the focus is getting trapped once you tab inside the theme color and it won't allow to come out of it due to that standard colors section also not accessible for keyboard users

Same issue for Highlighted color as well see attached screenshot, once you go inside color picker you wont able to come back outside using Shift-tab key.

![Image](https://github.com/user-attachments/assets/1159f144-e13d-4af8-9bee-bd6f2438848b)
